src/app/contents/iso639codes.js

```diff
@@ -0,0 +1,197 @@
+    ["ko", "Korean"],
+    ["ku", "Kurdish"],
+    ["kj", "Kwanyama, Kuanyama"],
+    ["la", "Latin"],
+    ["lb", "Luxembourgish, Letzeburgesch"],
+    ["lg", "Ganda"],
+    ["li", "Limburgish, Limburgan, Limburger"],
+    ["ln", "Lingala"],
+    ["lo", "Lao"],
+    ["lt", "Lithuanian"],
+    ["lu", "Luba-Katanga"],
+    ["lv", "Latvian"],
+    ["gv", "Manx"],
+    ["mk", "Macedonian"],
+    ["mg", "Malagasy"],
+    ["ms", "Malay"],
+    ["ml", "Malayalam"],
+    ["mt", "Maltese"],
+    ["mi", "MÄori"],
+    ["mr", "Marathi (MarÄá¹­hÄ«)"],
+    ["mh", "Marshallese"],
+    ["mn", "Mongolian"],
+    ["na", "Nauru"],
+    ["nv", "Navajo, Navaho"],
+    ["nb", "Norwegian BokmÃ¥l"],
+    ["nd", "North Ndebele"],
+    ["ne", "Nepali"],
+    ["ng", "Ndonga"],
+    ["nn", "Norwegian Nynorsk"],
+    ["no", "Norwegian"],
+    ["ii", "Nuosu"],
+    ["nr", "South Ndebele"],
+    ["oc", "Occitan"],
+    ["oj", "Ojibwe, Ojibwa"],
+    ["cu", "Old Church Slavonic, Church Slavic, Church Slavonic, Old Bulgarian, Old Slavonic"],
+    ["om", "Oromo"],
+    ["or", "Oriya"],
+    ["os", "Ossetian, Ossetic"],
+    ["pa", "Panjabi, Punjabi"],
+    ["pi", "PÄli"],
+    ["fa", "Persian (Farsi)"],
+    ["pl", "Polish"],
+    ["ps", "Pashto, Pushto"],
+    ["pt", "Portuguese"],
+    ["qu", "Quechua"],
+    ["rm", "Romansh"],
+    ["rn", "Kirundi"],
+    ["ro", "Romanian, [])"],
+    ["ru", "Russian"],
+    ["sa", "Sanskrit (Saá¹ská¹›ta)"],
+    ["sc", "Sardinian"],
+    ["sd", "Sindhi"],
+    ["se", "Northern Sami"],
+    ["sm", "Samoan"],
+    ["sg", "Sango"],
+    ["sr", "Serbian"],
+    ["gd", "Scottish Gaelic; Gaelic"],
+    ["sn", "Shona"],
+    ["si", "Sinhala, Sinhalese"],
+    ["sk", "Slovak"],
+    ["sl", "Slovene"],
+    ["so", "Somali"],
+    ["st", "Southern Sotho"],
+    ["az", "South Azerbaijani"],
+    ["es", "Spanish; Castilian"],
+    ["su", "Sundanese"],
+    ["sw", "Swahili"],
+    ["ss", "Swati"],
+    ["sv", "Swedish"],
+    ["ta", "Tamil"],
+    ["te", "Telugu"],
+    ["tg", "Tajik"],
+    ["th", "Thai"],
+    ["ti", "Tigrinya"],
+    ["bo", "Tibetan Standard, Tibetan, Central"],
+    ["tk", "Turkmen"],
+    ["tl", "Tagalog"],
+    ["tn", "Tswana"],
+    ["to", "Tonga (Tonga Islands)"],
+    ["tr", "Turkish"],
+    ["ts", "Tsonga"],
+    ["tt", "Tatar"],
+    ["tw", "Twi"],
+    ["ty", "Tahitian"],
+    ["ug", "Uyghur, Uighur"],
+    ["uk", "Ukrainian"],
+    ["ur", "Urdu"],
+    ["uz", "Uzbek"],
+    ["ve", "Venda"],
+    ["vi", "Vietnamese"],
+    ["vo", "VolapÃ¼k"],
+    ["wa", "Walloon"],
+    ["cy", "Welsh"],
+    ["wo", "Wolof"],
+    ["fy", "Western Frisian"],
+    ["xh", "Xhosa"],
+    ["yi", "Yiddish"],
+    ["yo", "Yoruba"],
+    ["za", "Zhuang, Chuang"],
+    ["zu", "Zulu"]
+];
+
+export default function getISO639Names(code){
+	var i;
+	for (i=0; i < langsNamesAndCodes.length; i++){
+		if (langsNamesAndCodes[i][0] == code) {
+			return langsNamesAndCodes[i][1];
+		}
+	}
+	throw "No language found with code: " + code;
+}
```

Comment:
I think we should use an external library so that we won't have to maintain the list ourselves. Also I imagine external libraries have the localized names of the languages, which will be useful when we fully localize the editor.

Reply:
@bfabio I agree with you on this. I just saw you were using a local file for language codes (src/app/contents/lang.js) and adopted the same approach.
I made a quick search and found some libraries. I think we could use this [https://www.npmjs.com/package/iso-639-1](https://www.npmjs.com/package/iso-639-1). Let me know how to proceed, thanks

Reply:
@ggrazia That's understandable. Ideally, that code should go as well as we adopt a library.

We already use `globalize` which depends on `cldrjs`. I think `cldrjs` can prove useful and on the plus size it's already installed and really comprehensive.

Reply:
Sorry, seeing this only now. I'm experimenting with calling external API to populate some drop-downs. The current 2 letter codes for languages and countries are in my opinion not very user friendly.
Here is a (seems reliable) API for countries https://datahelpdesk.worldbank.org/knowledgebase/articles/898590-country-api-queries.
I didn't do much research yet but I imagine we can find a similar one for languages.